partner_attribute_sst/views/res_partner_views.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='function']" position="after">
+                <field name="gender" widget="radio" />
+                <field name="date_of_birth" widget="date" />
+            </xpath>
+            <xpath expr="//field[@name='user_ids']" position="after">
+                <field name="fax" />
+            </xpath>
+            <xpath expr="//field[@name='category_id']" position="before">
+                <field name="occupation_id" groups="purchase.group_purchase_user" />
+                <field
+                    name="identification_type_id"
+                    groups="purchase.group_purchase_user"
+                />
+                <field
+                    name="identification_number"
+                    groups="purchase.group_purchase_user"
+                />
+            </xpath>
+            <xpath
+                expr="//field[@name='child_ids']/form//field[@name='mobile']"
+                position="after"
+            >
+                <field
+                    name="identification_type_id"
+                    groups="purchase.group_purchase_user"
+                />
+                <field
+                    name="identification_number"
+                    groups="purchase.group_purchase_user"
+                />
+            </xpath>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="res_partner_kanban_view">
```

Comment:
@kanda999 Can you please review and check if we should still keep all these changes in the kanban view?